infer/src/absint/TransferFunctions.mli

```diff
@@ -23,7 +23,7 @@ module type S = sig
   type instr
 
   val exec_instr : Domain.astate -> extras ProcData.t -> CFG.Node.t -> instr -> Domain.astate
-  (** {A} instr {A'}. [node] is the node of the current instruction *)
+  (** [A] instr [A']. [node] is the node of the current instruction *)
```

Comment:
These braces are semantically significant, better to put brackets around the entire triple.